.vscode/settings.json

```diff
@@ -21,6 +21,5 @@
     ],
     "workbench.colorCustomizations": {
         "activityBar.background": "#164e86"
-    },
-    "editor.formatOnSave": true
```

Comment:
This was causing vscode to change the formatting of my test code files. I really don't want a robot changing the format of my code. While in many cases that is okay, it is not okay in all cases. So I'd rather be responsible for the format myself.

Reply:
While I agree, the team collectively is moving to the vscode styled formatting (aka typescript's)...